doc/7/controllers/document/m-upsert/index.md

```diff
@@ -0,0 +1,62 @@
+---
+code: true
+type: page
+title: mUpsert
+description: Update documents
+---
+
+# mUpsert
+
+Applies partial changes to multiple documents. If a document doesn't already exist, a new document is created.
+
+You can set the `retryOnConflict` optional argument (with a retry count), to tell Kuzzle to retry the failing updates the specified amount of times before rejecting the request with an error.
+
+<br/>
+
+```js
+mUpsert(index, collection, documents, [options]);
+```
+
+| Argument     | Type                | Description                  |
+|--------------|---------------------|------------------------------|
+| `index`      | <pre>string</pre>   | Index name                   |
+| `collection` | <pre>string</pre>   | Collection name              |
+| `documents`  | <pre>object[]</pre> | Array of documents to update |
```

Comment:
Kuzzle expects a particular format for this argument (same as for single upserts). This should be detailed, especially considering that the sample code is missing the case where documents get created with default values.